### PR TITLE
Allow CCMs to get services and configmaps

### PR DESCRIPTION
**Pull request by rtheis**
IBM CCM has a load balancer service monitor that needs to get services in order
to properly monitor their overall health. It also needs to access cluster
configuration data via configmaps in the kube-system namespace. RBAC updates
for the former will likely always be required for the IBM CCM. The later could
be removed if the IBM CCM configuration was refactored.

Related: https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/97

### CODE REVIEW

**Review by Fedosin — APPROVED** [review]
/approve

**Review by elmiko — COMMENTED** [review]
thanks Richard!
/lgtm

**Review by jonesbr17 — APPROVED** [review]

**Comment by Fedosin** [comment]
/hold

until master is open.

**Comment by JoelSpeed** [comment]
Before we merge, can we get the PR description updated to explain why we need these permissions at the cluster scope

**Comment by rtheis** [comment]
@JoelSpeed PR description has been updated.

**Comment by JoelSpeed** [comment]
/lgtm
/hold cancel

Thanks @rtheis, I think this is good to go now

**Pull request merged**
